docs/DeveloperGuide.md

```diff
@@ -422,37 +422,37 @@ The following activity diagram shows what happens when a user executes a `findn`
+The proposed undo/redo mechanism is facilitated by `VersionedInTrack`. It extends `InTrack` with an undo/redo history, stored internally as an `intrackStateList` and `currentStatePointer`. Additionally, it implements the following operations:
 
-* `VersionedAddressBook#commit()` — Saves the current address book state in its history.
-* `VersionedAddressBook#undo()` — Restores the previous address book state from its history.
-* `VersionedAddressBook#redo()` — Restores a previously undone address book state from its history.
+* `VersionedInTrack#commit()` — Saves the current internship tracker state in its history.
+* `VersionedInTrack#undo()` — Restores the previous internship tracker state from its history.
+* `VersionedInTrack#redo()` — Restores a previously undone internship tracker state from its history.
 
-These operations are exposed in the `Model` interface as `Model#commitAddressBook()`, `Model#undoAddressBook()` and `Model#redoAddressBook()` respectively.
+These operations are exposed in the `Model` interface as `Model#commitInTrack()`, `Model#undoInTrack()` and `Model#redoInTrack()` respectively.
 
 Given below is an example usage scenario and how the undo/redo mechanism behaves at each step.
 
-Step 1. The user launches the application for the first time. The `VersionedAddressBook` will be initialized with the initial address book state, and the `currentStatePointer` pointing to that single address book state.
+Step 1. The user launches the application for the first time. The `VersionedInTrack` will be initialized with the initial internship tracker state, and the `currentStatePointer` pointing to that single internship tracker state.
 
 ![UndoRedoState0](images/UndoRedoState0.png)
 
-Step 2. The user executes `delete 5` command to delete the 5th internship in the address book. The `delete` command calls `Model#commitAddressBook()`, causing the modified state of the address book after the `delete 5` command executes to be saved in the `addressBookStateList`, and the `currentStatePointer` is shifted to the newly inserted address book state.
+Step 2. The user executes `delete 5` command to delete the 5th internship in the internship tracker. The `delete` command calls `Model#commitInTrack()`, causing the modified state of the internship tracker after the `delete 5` command executes to be saved in the `intrackStateList`, and the `currentStatePointer` is shifted to the newly inserted internship tracker state.
 
 ![UndoRedoState1](images/UndoRedoState1.png)
 
-Step 3. The user executes `add n/David …​` to add a new internship. The `add` command also calls `Model#commitAddressBook()`, causing another modified address book state to be saved into the `addressBookStateList`.
+Step 3. The user executes `add n/David …​` to add a new internship. The `add` command also calls `Model#commitInTrack()`, causing another modified internship tracker state to be saved into the `intrackStateList`.
```

Comment:
```suggestion
Step 3. The user executes `add c/Google …​` to add a new internship. The `add` command also calls `Model#commitInTrack()`, causing another modified internship tracker state to be saved into the `intrackStateList`.
```
Should be changed to `add c/...` as InTrack's add format is different.

Reply:
Yes, thank you for the reminder. The necessary changes have been pushed.